src/Parser.ts

```diff
@@ -45,11 +45,23 @@ export default class Parser {
             .shift() || this.lang.findDeclaration(node);
     }
 
+    findSourceFileDeclaration(
```

Comment:
I think we could probably rename this to `fileDeclaration` as we're not "finding" anything 🙂